src/views/Landing.vue

```diff
@@ -321,6 +546,19 @@ export default {
         grid-row-start: start;
         grid-row-end: end;
         grid-column-end: 1;
+        // TODO: DO something with this
```

Comment:
do what ?

Reply:
That's something we can use later but don't want to remove it.